docs/data-ingestion/vmagent.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Vmagent
+weight: 1
+sort: 1
+menu:
+  docs:
+    identifier: "vmagent"
+    parent: "data-ingestion"
+    weight: 1
+aliases:
+  - /data-ingestion/vmagent.html
+  - /data-ingestion/VMAgent.html
+---
+# Vmagent Setup
+`vmagent` is a small agent that can receive data in any format VictoriaMetrics Single and `vminsert` can ingest as well scrape Prometheus endpoints with the same job definitions.
+This section of the documentation only covers forwarding data from `vmagent` to another destination.
+For Information about `vmagent` as well as quickstart guide please refer to the [vmagent documentation](https://docs.victoriametrics.com/vmagent/)
+
+In any of the examples below you can add `insert/<tenant_id>/` to the URL path if you are sending metrics to vminsert.
+For example the remote write URL would change from
+
+```
+https://<victoriametrics_url>/prometheus/api/v1/write
+```
+
+to
+
+```
+https://<victoriametrics_url>/insert/<tenant_id>/prometheus/api/v1/write
+```
+
+
+## Sending data to VictoriaMetrics without authentication
+This requires setting the `-remote.url` flag in the command line arguments for vmagent
+
+```sh
+/path/to/vmagent -remoteWrite.url=https://<victoriametrics_url>:<victoriametrics_port>/api/v1/write
+```
+
+## Sending data to VictoriaMetrics with basic authentication
+
+```sh
+/path/to/vmagent -remoteWrite.url=https://<victoriametrics_url>:<victoriametrics_port>/api/v1/write -remoteWrite.basicAuth.username=<username> -remoteWrite.basicAuth.password=<password>
+```
+
+The basic authentication password can be placed in a file accessed via `-remoteWrite.passwordFile`.
+The file needs to be readable by the user `vmagent` is running as
```

Comment:
It should be -remoteWrite.basicAuth.passwordFile

Reply:
this was removed since I could not get this feature to work during testing